integration-tests/build_run_cluster.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+echo $DRUID_INTEGRATION_TEST_OVERRIDE_CONFIG_PATH
+
+export DIR=$(cd $(dirname $0) && pwd)
+export HADOOP_DOCKER_DIR=$DIR/../examples/quickstart/tutorial/hadoop/docker
+export DOCKERDIR=$DIR/docker
+export SERVICE_SUPERVISORDS_DIR=$DOCKERDIR/service-supervisords
+export ENVIRONMENT_CONFIGS_DIR=$DOCKERDIR/environment-configs
+export SHARED_DIR=${HOME}/shared
+export SUPERVISORDIR=/usr/lib/druid/conf
+export RESOURCEDIR=$DIR/src/test/resources
+
+# so docker IP addr will be known during docker build
+echo ${DOCKER_IP:=127.0.0.1} > $DOCKERDIR/docker_ip
+
+if !($DRUID_INTEGRATION_TEST_SKIP_BUILD_DOCKER); then
```

Comment:
Why the split DRUID_INTEGRATION_TEST_SKIP_BUILD_DOCKER and DRUID_INTEGRATION_TEST_SKIP_RUN_DOCKER?
I don't think we should ever want to BUILD but not RUN. I am also not really sure if having not BUILD and run is that useful. If your cluster is not running then you are not sure what state it is in, might as well be better to build and start fresh.

Reply:
usually if you develop tests you need to build docker container only once. You dont need to do this more then once because you do not develop any of things that is in container.
You can run "build" flow without "running tests" in case if do this on CI and you want to build once container before runnning all types of integration tests. 
Your build flow will be next:
1. build docker container 
2. run parallel:
2.1 s3 tests on builded container
2.2 azure tests on builded container
2.3 gcs tests on builded container
2.4 etc

Reply:
Sounds good.

Reply:
I think we can keep skip.build and skip.start. 
We can then get rid of skip.stop.

Reply:
fixed,
added info about docker.build.skip into readme
removed docker.stop.skip variable